src/qt/dbb_gui.cpp

```diff
@@ -1052,7 +1052,17 @@ void DBBDaemonGui::eraseClicked()
 void DBBDaemonGui::resetU2F()
 {
     DBB::LogPrint("Request U2F reset...\n", "");
-    if (executeCommandWrapper("{\"reset\":\"U2F\"}", DBB_PROCESS_INFOLAYER_STYLE_TOUCHBUTTON, [this](const std::string& cmdOut, dbb_cmd_execution_status_t status) {
+
+    std::string cmd("{\"reset\":\"U2F\"}");
+    QString version = this->ui->versionLabel->text();
```

Comment:
I think we should not read UI labels for this... Let me check how we could do this more stable...